head/src/main/kotlin/io/qalipsis/core/head/report/DataSeriesServiceImpl.kt

```diff
@@ -196,4 +203,27 @@ internal class DataSeriesServiceImpl(
         )
     }
 
+    override suspend fun refresh() {
+        orchestrationCoroutineScope.launch {
+            val dataSeriesEntities = dataSeriesRepository.findAll()
+            val size = dataSeriesEntities.count()
+            logger.trace { "Updating the queries of $size retrieved data series" }
+            dataSeriesEntities.collect { dataSeriesEntity ->
```

Comment:
Add a try catch in case something wrong happens, and also add logs for success.
If you use pagination, log the current state done of the total.